lib/clixon/clixon_xml_map.h

```diff
@@ -40,6 +40,15 @@
 #ifndef _CLIXON_XML_MAP_H_
 #define _CLIXON_XML_MAP_H_
 
+/*
+ * Maximum number of supported bit pisitions in YANG "bits" data 
+ * type. As defined in RFC7950 (section 9.7.4.2.) the position 
+ * value must be in the range 0 to 4294967295. But who needs
+ * that much bit positions? (To set bit 4'294'967'295 it would be
+ * necessary to tranfer 4294967295/8 = 536'870'911 bytes!)
+ */
+#define CLIXON_BITS_POS_MAX 1024
```

Comment:
One could consider using a config option here. That said, I think it is best that it being a constant for now. One consequence in a future change would be that one needs to pass clixon_handle h to all functions.